src/WeedStore.js

```diff
@@ -20,7 +23,7 @@ export default class WeedStore {
   setNewData = () => {
     let keys = Object.keys(this.allData)
     let strainsArray = keys.map(key => {
-      return {name: key, ...this.allData[key]}
+      return {name: key, ...this.allData[key], favorite: false}
```

Comment:
added favorite property to each strain so we can keep track of whether it's been favorited or not.